modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseLocalServiceImpl.java

```diff
@@ -60,6 +62,75 @@
 public class CommerceInventoryWarehouseLocalServiceImpl
 	extends CommerceInventoryWarehouseLocalServiceBaseImpl {
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceInventoryWarehouse addCommerceInventoryWarehouse(
```

Comment:
@CrescenzoRegaLF here you added a new method instead of updating the existing one.
The "name" now is a localized field, doesn't make senso to keep the old method that store a plain String.
You should also update all usages of the old method with the new one.